pkgs/games/domination/default.nix

```diff
@@ -83,6 +84,11 @@ in stdenv.mkDerivation {
     runHook postInstall
   '';
 
+  preFixup = ''
+    # remove extra metadata files for jar files which break patchJavaArchivesHook
```

Comment:
Isn't it stripJavaArchivesHook (not patch*)?

Reply:
Yeah, sorry, I always mix the two words up...